### PR TITLE
Add example for pango text attributes

### PR DESCRIPTION
**Pull request by EPashkin**
cc @GuillaumeGomez

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Please add explanations about what the code is doing (it's examples after all). :)

**Comment by EPashkin** [comment]
Updated

File: src/bin/pango_attributes.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+//! # Pango text attributes
+//!
+//! This sample demonstrates how to use various attributes (strikeout, background color, etc. ) on labels text
```
Comment (GuillaumeGomez) [inline src/bin/pango_attributes.rs]: Missing dot at the end of the line. Also, can you keep the line's length under 100 column please?

**Comment by GuillaumeGomez** [comment]
Not much comments, but considering the "complexity" of the example, I think just the description at the beginning of the file is enough.

**Comment by EPashkin** [comment]
Updated

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**